lib/MongoDB/MongoClient.pm

```diff
@@ -1624,7 +1624,8 @@ sub send_retryable_write_op {
         my $retry_link = $self->{_topology}->get_writable_link;
 
         # Rare chance that the new link is not retryable
```

Comment:
Comment is now mildly confusing given extra error check.

Reply:
Oh oops I did have a comment there about the extra check, must have got caught with a debugging cleanup...